docs/developers-clear-containers-install.md

```diff
@@ -102,7 +102,7 @@ $ sudo -E apt-get install -y qemu-lite
 ```bash
 $ export clear_release=$(curl -sL https://download.clearlinux.org/latest)
 $ export cc_img_path="/usr/share/clear-containers"
-$ export kernel_clear_release=12760
+$ export kernel_clear_release=demos
```

Comment:
@chavafg  also, the demos kernel must be semaphore specific, isn't it ? for developers is ok to use a newest kernel.